tests/spacepy_testing.py

```diff
@@ -31,9 +31,10 @@ def add_build_to_path():
     """
     # Prioritize version-specific path; py2 tends to be version-specific
     # and py3 tends to use just "lib". But only use first-matching.
-    for pth in ('lib.{0}-{1}.{2}'.format(sysconfig.get_platform(),
+    for pth in ('lib',  # Prepending, so add low-priority paths first.
+                'lib.{0}-{1}.{2}'.format(sysconfig.get_platform(),
                                          *sys.version_info[:2]),
-                'lib'):
+    ):
```

Comment:
See other comment about indent level. I think line 37 should be indented to the same level as line 35.

Reply:
and yep. tests are running on the push.